packages/react-components/react-color-picker-preview/library/README.md

```diff
@@ -3,3 +3,47 @@
 **React Color Picker components for [Fluent UI React](https://react.fluentui.dev/)**
 
 These are not production-ready components and **should never be used in product**. This space is useful for testing new components whose APIs might change before final release.
+
+The ColorPicker is used to browse through and select colors.
+By default, it lets people navigate through colors on a color spectrum; or specify a color in either Red-Green-Blue (RGB); or alpha color code; or Hexadecimal textboxes.
```

Comment:
Does the color picker currently support the RGB color format? My understanding was that it only supports HSV/HSL color formats.

Reply:
Yes, ColorPicker itself works with HSV. But input boxes (which I'll add in stories) will support RGB and HEX. Here's an example of how it looks in Office:
![image](https://github.com/user-attachments/assets/96dbc50c-c695-4222-bb93-7ba39805b09e)

Reply:
Though I think it's good to point out this behavior in the Spec. Thanks for catching that